cdk/lib/__snapshots__/prism.test.ts.snap

```diff
@@ -573,7 +573,7 @@ dpkg -i /prism/prism.deb",
       },
       "Type": "AWS::IAM::Role",
     },
-    "LoadBalancer": Object {
+    "LoadBalancerBE9EEC3A": Object {
```

Comment:
@akash1810 - I think we'll need to set `existingLogicalId` on the load balancer. I'm a bit confused as to how this id has remained static so far, as `overrideId` doesn't currently appear to be set 😕

Reply:
Argh! Good spot! Will add `existingLogicalId`.

I think this is the result of https://github.com/guardian/cdk/pull/440. Looking at the [removed tests](https://github.com/guardian/cdk/commit/0d0eba739e97c18baa411f3f4eb58b6fa26727bb#diff-ddcd012c9ad844faf4c3b865a658f5e3e600d62ad88228ca022798def7c56c9dL33), it looks like if `migratedFromCloudFormation` was true on the `GuStack`, the construct's logicalId was not auto-generated; the value passed into the constructor was used.

I think this is another +1 for https://github.com/guardian/cdk/pull/364 and encapsulate this logic in one place.

I'd also have hoped to see [the message](https://github.com/guardian/cdk/blob/3eefaaab116b101498151ab8b9e94dc996d9eb21/src/constructs/core/migrating.ts#L70) in the console, but then I realised I've been using Jest locally which sets NODE_ENV to test - oops https://github.com/guardian/cdk/pull/397!